cardano-byron-proxy.cabal

```diff
@@ -136,6 +136,8 @@ executable cardano-byron-proxy
                        -Wincomplete-uni-patterns
 
 executable cddl-test
+  -- Don't build until somebody comes along and maintains this
+  buildable:           False
```

Comment:
Agreed

Reply:
`cddl-test` is using the `ImmDB` directly, while that module is only meant for internal use, so things will often break.